packages/design-system/src/components/analytics/index.ts

```diff
@@ -1 +1 @@
-export { default as sendAnalytics } from './SendAnalytics';
+export { default as sendAnalytics } from './sendAnalytics';
```

Comment:
[nitpick] does this need to stay `SendAnalytics` to match the file name casing?

Reply:
Whoops, that was left over from a time when I thought I'd change the case of some files, and it was a terrible idea.